.github/workflows/build.yml

```
@@ -367,3 +367,64 @@ jobs:
           name: e2e-report
           path: testing/e2e/playwright-report/
           retention-days: 30
+
+  veracode:
+    name: Zip and Upload files to Veracode
+    runs-on: ubuntu-latest
+    # if: github.ref == 'refs/heads/main'
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Zip files for veracode
+        shell: bash
+        run: |
+          # Define the source directories and output file
+          sourceDirs=("apps" "packages")
+          outputFile="itwui-veracode.zip"
+
+          # Define the paths to exclude
+          excludedPaths=(
+            "**/node_modules/*"
+            ".git/*"
+            "**/backstop/*"
+            '**/dist/*'
+            '**/build/*'
+            "**/cypress*/*"
+            "**/.turbo/*"
+            "**/.astro/*"
+            "**/.next/*"
```

We can reduce the number of excluded paths, since we don't run `install` or `build`.

```suggestion
            ".git/*"
            "**/backstop/*"
            "**/cypress*/*"
```